src/init.c

```diff
@@ -298,6 +298,9 @@ shmem_internal_init(int tl_requested, int *tl_provided)
             printf("SMA_INFO                %s\n",
                    (NULL != shmem_util_getenv_str("INFO")) ? "Set" : "Not set");
             printf("\tIf set, print this help message at startup\n");
+            printf("SMA_OFI_ATOMIC_CHECKS_WARN  %s\n",
```

Comment:
In the interest of closing out this PR, why don't we file an issue to add an info function to the transport interface and drop this for now?